.github/workflows/python-app.yml

```diff
@@ -62,4 +50,9 @@
      - name: flake8 linter
        run:  |
          source ./.venv/bin/activate
-         python -m flake8 --count --max-complexity 10 --ignore E501,W503 --show-source --statistics --exclude ./.venv .
\ No newline at end of file
+         python -m flake8 --count --max-complexity 10 --ignore E501,W503 --show-source --statistics --exclude ./.venv .
+     - name: Run mypy
+       run: |
+         source ./.venv/bin/activate       
+         pip install mypy
+         mypy --install-types --non-interactive --namespace-packages
```

Comment:
This is all we're doing to run it.

Reply:
I tried a couple of pre-existing actions to get it to go and had no luck so just ran it manually on the command-line.  The actions supposedly annotate the PR with errors, but otherwise just running it likes this seems to do the trick just as well.